tests/general/LicenseCheckBearTest.py

```diff
@@ -0,0 +1,78 @@
+from queue import Queue
+import os.path
+
+from bears.general.LicenseCheckBear import LicenseCheckBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+
+
+def get_testfile_path(name):
+    return os.path.join(os.path.dirname(__file__),
+                        'licensecheck_test_files',
+                        name)
+
+
+def load_testfile(name):
+    with open(get_testfile_path(name)) as fl:
+        contents = fl.read()
+
+    return contents
+
+
+@generate_skip_decorator(LicenseCheckBear)
+class LicenseCheckBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.uut = LicenseCheckBear(Section('name'), Queue())
+
+    def test_license(self):
+        file_contents = [load_testfile(
+            get_testfile_path('apache_license_2.py'))]
+        test_file_path = get_testfile_path('apache_license_2.py')
+        self.check_results(
+            self.uut,
+            file_contents,
+            [Result.from_values('LicenseCheckBear',
+                                test_file_path + ': Apache (v2.0)',
```

Comment:
Unlike other bears that check validity,
> This bear attempts to determine the license that applies to each file passed to
    it, by searching the start of the file for text belonging to various
    licenses.

It will return a result all the time to determine not just the license but also the type of license.